rest_framework-stubs/fields.pyi

```diff
@@ -538,8 +546,8 @@ class ChoiceField(Field[str, Union[str, int, Tuple[Union[str, int], Union[str, i
         read_only: bool = ...,
         write_only: bool = ...,
         required: bool = ...,
-        default: Union[str, int, Callable[[], str], Callable[[], int]] = ...,
-        initial: Union[str, int, Callable[[], str], Callable[[], int]] = ...,
+        default: Union[_Empty, str, int, Callable[[], str], Callable[[], int]] = ...,
```

Comment:
not using the type alias here?

Reply:
not sure what's the point to write aliases for such cases, we can't use _DefaultInitial alias in this case

Reply:
@kalekseev I had the same question and I don't understand the answer.

I think we **can** use
```python
default: _DefaultInitial[Union[str, int]] = ...,
initial: _DefaultInitial[Union[str, int]] = ...,
```

I opened PR #152, please respond there if you see any issues with my approach.

Reply:
@intgr I don't remember my thoughts on that but most probably I just missed the ability to pass union as generic param, so your pr is on point